server/memphis_handlers_stations.go

```diff
@@ -1076,13 +1076,17 @@ func (sh StationsHandler) GetMessageDetails(c *gin.Context) {
 		return
 	}
 
-	connectionIdHeader := hdr["connectionId"]
-	producedByHeader := strings.ToLower(hdr["producedBy"])
+	connectionIdHeader := hdr["$memphisconnectionId"]
```

Comment:
I write it here but this is regarding the function resendPoisonMessage - this function sends a message, currently it sends the payload only, you need to send headers as well